### PR TITLE
Fix links for EN docs in README

### LINKED ISSUE #1676: Documentation link in readme is 404
http://docs.phinx.org/en/latest -> 404

### PR DESCRIPTION
**Pull request by MasterOdin**
Fixes #1676

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 Phinx makes it ridiculously easy to manage the database migrations for your PHP app. In less than 5 minutes, you can install Phinx and create your first database migration. Phinx is just about migrations without all the bloat of a database ORM system or framework.
 
-**Check out [docs.phinx.org/en/latest/](http://docs.phinx.org/en/latest/) ([EN](http://docs.phinx.org/en/latest/), [ZH](https://tsy12321.gitbooks.io/phinx-doc/)) for the comprehensive documentation.**
+**Check out [book.cakephp.org/4/en/phinx.html](https://book.cakephp.org/4/en/phinx.html) ([EN](https://book.cakephp.org/4/en/phinx.html), [ZH](https://tsy12321.gitbooks.io/phinx-doc/)) for the comprehensive documentation.**
```
Comment (saeideng) [inline README.md]: this links content will remove soon
and will move to `https://book.cakephp.org/phinx/0` or for latest `https://book.cakephp.org/phinx`

Comment (saeideng) [reply README.md]: just use the  `https://book.cakephp.org/phinx` IMO
this link will fix soon

Comment (saeideng) [reply README.md]: you can use the `https://book.cakephp.org/phinx` now 
now it works well

Comment (MasterOdin) [reply README.md]: Done

**Pull request merged**